stdlib/src/sys/ffi/utils.mojo

```diff
@@ -17,8 +17,8 @@ from memory import UnsafePointer
 
 from utils import StringRef
 
-from .info import os_is_linux, os_is_windows, is_64bit, os_is_macos
-from .intrinsics import _mlirtype_is_eq
+from sys.info import os_is_linux, os_is_windows, is_64bit, os_is_macos
+from sys.intrinsics import _mlirtype_is_eq
 
 from sys._libc import dlerror, dlopen, dlclose, dlsym
```

Comment:
@martinvuyk what about the `sys._libc` module? are they going to merge at some point?

Reply:
Hi @msaelices, yes I imagine they will merge at some point. There is also some future cleanup to switch `c_int` to use `C.int` and other such variables (if they decide to merge this PR at all).